draft-ietf-tls-tls13.md

```diff
@@ -1553,8 +1553,12 @@ Ephemeral Secret (ES): A secret which is derived from fresh (EC)DHE
 Static Secret (SS): A secret which may be derived from static or
    semi-static keying material, such as a pre-shared key or the
    server's semi-static (EC)DH share.
+   
+Master Secret (SS): A secret derived from both the static 
```

Comment:
This should be "MS" here instead of "SS". That said, I think I'd prefer just dropping this parenthetical as it's not abbreviated like that anywhere like the other two are.